extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/security/CSRF.java

```diff
@@ -0,0 +1,150 @@
+package io.quarkus.vertx.http.security;
+
+import java.time.Duration;
+import java.util.Set;
+
+/**
+ * This class provides a way to configure the Cross-Site Request Forgery (CSRF) prevention.
+ */
+public interface CSRF {
```

Comment:
@michalvavrik That should also have an experimental status for now, similar to other such interfaces added earlier ? Please check the cors one too

Reply:
CORS is ok, I forgot here, sorry